tools/mocha-runner/src/interface.ts

```diff
@@ -110,14 +174,43 @@ function customBDDInterface(suite: Mocha.Suite) {
       it.only = function (title: string, fn: Mocha.TestFunction) {
         return common.test.only(
           mocha,
-          (context['it'] as typeof it)(title, fn, true)
+          (context['it'] as unknown as typeof it)(title, fn, true)
         );
       };
 
       it.skip = function (title: string) {
         return context['it'](title);
       };
 
+      it.only = function (title: string, fn: Mocha.TestFunction) {
+        return common.test.only(
+          mocha,
+          (context['it'] as unknown as typeof it)(title, fn, true)
+        );
+      };
+
+      it.skip = function (title: string) {
+        return context['it'](title);
+      };
+
+      function wrapDeflake(
+        func: Function
+      ): (repeats: number, title: string, fn: Mocha.AsyncFunc) => void {
+        return (repeats: number, title: string, fn: Mocha.AsyncFunc): void => {
+          for (let i = 1; i <= repeats; i++) {
+            (context['describe'] as unknown as typeof describe).withDebugLogs(
+              `deflake ${title}`,
+              () => {
+                func(`${i}/${title}`, fn);
+              }
+            );
+          }
+        };
+      }
+
+      it.deflake = wrapDeflake(it);
```

Comment:
remove

Reply:
I would prefer to keep this as it's easier for me to use locally. EsLint tooling is provided in the PR to prevent submitting it to main.
For the offline question about `How does it work with Expectation`:
The short answer it it does not work on the verifying part and it can't as we need to provide a wrapper `describe` to use Before/After hooks and that will always put spaces in the `fullTestName`.

Reply:
I disagree, it looks like another way to do the same thing that is not much more convenient than using the env variable and `.only`. I would prefer not to double the number of it helpers for this feature.

Reply:
Consider this: You have a test that you know is flaky or you wanna stress test it.
If we have only only `env` we have to find a pattern that matches the test usually `[<fileName>] * <testName>`,
but you have to remember a variable name and write it in Terminal, also due to the way our test are named that may hit other test as well. 
Or you can prepend the test with the helper that is auto-completed by most IDEs and has TsDoc explaining what it does + the test out recompile and build so you don't need an extra step.

I think where I am going is - I don't see the problem with having both options available.